api/src/pcapi/scripts/subscription/dms/handle_single_inactive_application.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+from sqlalchemy.orm import exc as sqla_exc
+
+from pcapi import settings
+from pcapi.connectors.dms import api as dms_api
+from pcapi.connectors.dms import exceptions as dms_exceptions
+from pcapi.connectors.dms import models as dms_models
+from pcapi.core.fraud import models as fraud_models
+from pcapi.repository import repository
+
+
+logger = logging.getLogger(__name__)
+
+
+def handle_single_inactive_dms_application(application_number: int) -> None:
+    logger.info("[DMS] Handling single inactive application %d", application_number)
+    marked_applications_count = 0
+    draft_application = dms_api.DMSGraphQLClient().get_single_application_details(application_number)
+
+    try:
+        _mark_without_continuation_a_draft_application(draft_application)
+        _mark_cancel_dms_fraud_check(draft_application.number, draft_application.profile.email)
+        marked_applications_count += 1
+    except (dms_exceptions.DmsGraphQLApiException, Exception):  # pylint: disable=broad-except
+        logger.exception("[DMS] Could not mark application %s without continuation", draft_application.number)
+
+
+def _mark_without_continuation_a_draft_application(dms_application: dms_models.DmsApplicationResponse) -> None:
+    dms_api.DMSGraphQLClient().make_on_going(
+        dms_application.id, settings.DMS_ENROLLMENT_INSTRUCTOR, disable_notification=True
+    )
+    dms_api.DMSGraphQLClient().mark_without_continuation(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏De mémoire, le dossier est bloqué parce que `make_on_going` a marché et `mark_without_continuation` a KO.
Tu peux mettre un petit log après chaque pour voir où ça passe / bloque, si tu veux